front-end/public/index.html

```diff
@@ -2,14 +2,15 @@
 <html lang="en">
```

Comment:
Also wouldn't change the index.html, this is just automatically made by `create-react-app` and we won't render it anyway.

Reply:
As it is, index.html is indeed being rendered. That's where the page title and favicon goes. I changed so it wouldn't say React stuff and use React's logo. Don't know how to build a React app without rendering index.html but if that's the case we can remove this file when we get there.

Reply:
Sorry I meant like visually you're only seeing what `App.js` is rendering. I don't think we can remove the file but I just never really see it customized is all.